airflow/models/dagrun.py

```diff
@@ -1387,7 +1388,7 @@ def schedule_tis(
     @provide_session
     def get_log_template(self, *, session: Session = NEW_SESSION) -> LogTemplate:
         if self.log_template_id is None:  # DagRun created before LogTemplate introduction.
-            template = session.scalar(select(LogTemplate).order_by(LogTemplate.id))
+            template = session.scalar(select(LogTemplate).order_by(LogTemplate.id).limit(1))
```

Comment:
Meaningless since `id` is the primary key.

Reply:
Not really, the query for this condition is ```SELECT * FROM log ORDER BY log.id``` and result filtration happen on the client, nor DB backend.

Reply:
Oh this is `ORDER BY` not `WHERE`, I misread.